src/sentry/monitors/consumers/check_in.py

```diff
@@ -175,6 +175,12 @@ def _process_message(wrapper: Dict) -> None:
                 if duration is not None:
                     date_added -= datetime.timedelta(milliseconds=duration)
 
+                expected_time = None
+                if monitor_environment.last_checkin:
+                    expected_time = monitor.get_next_scheduled_checkin_without_margin(
```

Comment:
After this PR can you flip this, so it's just `get_next_scheduled_checkin` and then for cases where we included the margin right now have a named arg like `with_margin=True`.

Should make things a lot more clear.

Reply:
ahh yes, that is good